tests/unit/src/main/scala/tests/BaseLspSuite.scala

```diff
@@ -49,6 +51,12 @@ abstract class BaseLspSuite(
 
   protected def initializationOptions: Option[InitializationOptions] = None
 
+  private val virtualDocTag = new Tag("UseVirtualDocs")
```

Comment:
I think the tag approach is ok with a small modification for removing the var.

Reply:
Do you mean removing the `val virtualDocTag`?  If so - where would you put it - in a companion object?  Or do you mean remove the `val useVirtualsDoc` var?

Reply:
I meant the var, but if we still need it then it's ok.